src/uwtools/cli/atparse_to_jinja2.py

```diff
@@ -15,6 +16,7 @@ def main() -> None:
     Main entry point.
     """
     args = parse_args(sys.argv[1:])
+    setup_logging(quiet=args.quiet, verbose=args.verbose)
```

Comment:
Here's the first example of logging setup immediately following command-line argument parsing. Others follow.